db/migrate/20200316193520_create_consultation_tables.rb

```diff
@@ -0,0 +1,38 @@
+class CreateConsultationTables < ActiveRecord::Migration[5.1]
```

Comment:
This file is generated in Step 1 and populated by a developer in Step 2 of the [Creating a new data source tutorial](https://github.com/metridoc/metridoc-rails/wiki/Creating-a-new-data-type).